SOP/SOP-mdc.adoc

```diff
@@ -0,0 +1,81 @@
+ifdef::env-github[]
+:status:
+:tip-caption: :bulb:
+:note-caption: :information_source:
+:important-caption: :heavy_exclamation_mark:
```

Comment:
Missing: 
- Reference Articles
- Prometheus Alerts Procedures
- Validate
-- Installation

See here: https://github.com/aerogear/unifiedpush-operator/blob/master/SOP/SOP-push.adoc

Reply:
My reasoning for not including the reference article's here is because we include them in the SOP for the operator. The first thin we ask the user to do in the mdc SOP is to check the operator SOP so they would see it there. Do we need to duplicate it?

Reply:
Same comment as above about the Prometheus Alerts Procedures, seems to be very similar to the overview as it contain text which is very similar to text which appears in other sections.

Reply:
The text in the push SOP for Premehteus procedures is the following 
```

== Prometheus Alerts Procedures

IMPORTANT: Before checking any of following steps see if the operator pod is running successfully. More info: link:./SOP-operator.adoc[UnifiedPushOperatorDown]

TIP: You can save the logs by running `oc logs <podname> > <filename>.log`. The logs can provided to you useful information in order to identify the root cause of the issue as they are required information to be send to its maintainers in the case it need to be checked by them.
```
To me this doesn't sound like procedure but some general information that we could include elsewhere.

Reply:
The SOP's need to be done thinking in who has no idea over the project and all. So, it is important IMHO.